skaffold/src/main/kotlin/com/google/container/tools/skaffold/run/SkaffoldCommandLineState.kt

```diff
@@ -60,6 +64,24 @@ class SkaffoldCommandLineState(
             throw ExecutionException(message("skaffold.no.file.selected.error"))
         }
 
+        //call skaffold version --- look at createProcess function, the command list lets you know if skaffold isn't on the system by process of elimination
+        if (Paths.get("skaffold").exists()) {
+            val testSkaffoldCommand = Paths.get("skaffold").toString() + " " + (executionMode.modeFlag)
```

Comment:
one reason the build is failing is because this line is over 100 chars. you can see this by running `./gradlew check`. these currently have to be fixed manually because `./gradlew spotlessApply` does not yet fix line length violations

Reply:
by "manually" I mean you can run a reformat in intellij: code -> reformat code